### PR TITLE
Updates + corrects wallet_quickstart_fr.md

### PR DESCRIPTION
**Pull request by HarryMacfinned**
Corrects a bunch of typos + spelling.
Also updates the BC size.

### CODE REVIEW

File: monero-outreach-docs/translations/fr/storyboard/wallet_quickstart_fr.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Portefueille Monero, démarrage rapide
-*22/04/2019*  
-Almutasim  
+# Portefeuille Monero, démarrage rapide
+*22/04/2019* Almutasim
```
Comment (erciccione) [inline monero-outreach-docs/translations/fr/storyboard/wallet_quickstart_fr.md]: Please keep the original format of the document.

File: monero-outreach-docs/translations/fr/storyboard/wallet_quickstart_fr.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Portefueille Monero, démarrage rapide
-*22/04/2019*  
-Almutasim  
+# Portefeuille Monero, démarrage rapide
+*22/04/2019* Almutasim  
+*28/07/2019* mis à jour
```
Comment (erciccione) [inline monero-outreach-docs/translations/fr/storyboard/wallet_quickstart_fr.md]: I don't thin the other documents have a note after the date. I think `mis à jour` should be removed, but better wait for @lh1008 's review

**Comment by erciccione** [comment]
ping @el00ruobuob for review.

File: monero-outreach-docs/translations/fr/storyboard/wallet_quickstart_fr.md (hunk 2 above)
Comment (lh1008) [reply monero-outreach-docs/translations/fr/storyboard/wallet_quickstart_fr.md]: Yes please remove `mis à jour`.

File: monero-outreach-docs/translations/fr/storyboard/wallet_quickstart_fr.md (hunk 1 above)
Comment (lh1008) [reply monero-outreach-docs/translations/fr/storyboard/wallet_quickstart_fr.md]: Yes please keep the original format of the document.

**Comment by el00ruobuob** [comment]
Reviewed

**Comment by HarryMacfinned** [comment]
Thanks for all your remarks and review and apologies for the wrong doing.

**Pull request merged**